tests/gocase/unit/introspection/replication_test.go

```diff
@@ -0,0 +1,349 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package introspection
```

Comment:
I suggest this case should be under `gocase/integration/replication/replication_test.go`.

Reply:
okay, will move it.

Reply:
@git-hulk I think you still hold the test under:

* `gocase/unit`

instead of:

* `gocase/integration`.

Reply:
@tisonkun it's in `[tests/gocase/unit/replication/replication_test.go](https://github.com/apache/incubator-kvrocks/pull/997/files#diff-99ea2797442d2eaf01c9416f031da8c8089fa5725188e2b4f1a961d4572a9f7a)` indeed, do you mean should move into the `integration` dir?

Reply:
Yes. Replication tests originality lays under tcl/tests/integration.

Reply:
Oops, I misremembered the dir when migrating.